internal/curves/curve_test.go

```diff
@@ -205,6 +205,65 @@ func TestFunctionCurveAverage(t *testing.T) {
 	assert.Equal(t, 127, result)
 }
 
+func TestFunctionCurveDelta (t *testing.T) {
+	// GIVEN
+	temp1 := 20000.0
+	temp2 := 40000.0
+
+	s1 := MockSensor{
+		ID:        "ambient_sensor",
+		Name:      "sensor_ambient",
+		MovingAvg: temp1,
+	}
+	sensors.SensorMap[s1.GetId()] = &s1
+
+	s2 := MockSensor{
+		ID:        "water_sensor",
+		Name:      "sensor_water",
+		MovingAvg: temp2,
+	}
+	sensors.SensorMap[s2.GetId()] = &s2
+
+	curve1 :=  createLinearCurveConfig(
+		"case_fan_front",
+		s1.GetId(),
+		18,
+		60,
```

Comment:
I don't really understand how you choose those values, but it does work as expected so... :+1:  :sweat_smile:

Reply:
I don't remember, I think I as testing different numbers and just checked those in, lol.